corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/utils.js

```diff
@@ -92,6 +97,7 @@ hqDefine("cloudcare/js/formplayer/utils/utils", function () {
     };
 
     Utils.getDisplayOptionsKey = function () {
+        // TODO: use hqRequire
```

Comment:
Was this one missed or does it need to be swapped later for some reason?

Reply:
This got cleaned up later, in https://github.com/dimagi/commcare-hq/pull/32233/commits/873dc93cedd5a43410c45e3e6a266657f392fccd